bHY6yh-de2065/cod-yyyiou_bHY6yh-de2065.xml

```diff
@@ -0,0 +1,210 @@
+               <date when="2023-05-31">May 31, 2023</date>
+            </edition>
+         </editionStmt>
+         <publicationStmt>
+            <authority>SCTA</authority>
+            <availability status="free">
+               <p>Published under a <ref target="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</ref>
+               </p>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <listWit>
+               <witness xml:id="C" n="cod-yyyiou">Vatican, Biblioteca Apostolica Vaticana, Ross. 429 (IX.119)</witness>
+            </listWit>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+         <editorialDecl>
+            <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+       guidelines for a diplomatic edition.</p>
+         </editorialDecl>
+      </encodingDesc>
+      <revisionDesc status="draft">
+         <listChange>
+            <change when="2023-05-31" status="draft" n="0.0.0">
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+            <pb ed="#V" n="3-r"/>
+            <cb ed="#V" n="b"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-de2065"><!-- p1l1-r8 -->
+            <head xml:id="bHY6yh-de2065-Hd1e108">Rubrica 8</head>
+            <head xml:id="bHY6yh-de2065-Hd1e111" type="question-title">De iustitia et iudicio Dei</head>
+            <p xml:id="bHY6yh-de2065-d1e114">De charitate seu dilectione et odiositate dei ad hominem. Rubrica. 8.
+    </p>
+            <p xml:id="bHY6yh-de2065-d1e121">
+                <ref xml:id="bHY6yh-de2065-Rd1e118">IOb ystoria capitulo XVIo</ref> <quote xml:id="bHY6yh-de2065-Qd1e125">
+                <lb ed="#0"/>Multo minora ex
+                <lb ed="#0"/>igeris 
+                a deo qua meretur in
+                <lb ed="#0"/>iquitas
+                 tua. </quote>
+            </p>
+            <p xml:id="bHY6yh-de2065-d1e133">
+                <ref xml:id="bHY6yh-de2065-Rd1e132">@Psamista pslamus V</ref> <quote xml:id="bHY6yh-de2065-Qd1e137">O
+                <lb ed="#0"/>disti 
+                omnes qui operantur iniquitatem
+                <lb ed="#0"/> perdes omnes quilequitur mendicans</quote>
+            </p>
+            <p xml:id="bHY6yh-de2065-d1e144"> 
+      <quote xml:id="bHY6yh-de2065-Qd1e146">Virum 
+               sanguinum et dolosum abominabitur dominus. </quote> 
+            </p>
+            <p xml:id="bHY6yh-de2065-d1e152"> 
+       <ref xml:id="bHY6yh-de2065-Rd1e151">@Idem XXXIII</ref> <quote xml:id="bHY6yh-de2065-Qd1e154">Oculi 
```

Comment:
Are they using an "@" in your record? What does that represent?

Reply:
maybe the `@` are meant to be stand in's for the pilcrow. If so they need to be converted to pilcrows. Also I'm not sure `@` is the best symbol to use for find and replace, since it get sometimes gets used elsewhere and we risk accidental replacements. I use the π symbol, which on my keyboard is just alt+p

Reply:
Yes i am using @s for the pilcrow, but i forgot to replace them before i committed that file, sorry